manifests/0000_90_kube-scheduler-operator_04_servicemonitor-scheduler.yaml

```diff
@@ -85,6 +85,8 @@ spec:
     tlsConfig:
       caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
       serverName: scheduler.openshift-kube-scheduler.svc
+      certFile: /etc/prometheus/secrets/metrics-client-certs/tls.crt
```

Comment:
Should `bearerTokenFile` be removed alongside?

Reply:
Good question. When everything works, bearerTokenFile should be removed for service monitor. But as it turned out, kube-rbac-proxy is not yet coded/configured properly to handle client certificate authentication yet. I am in the process of fixing it. Original PR can go in without harm. But removing bearerTokenFile will have to be done separately after the fix in kube-rbac-proxy.